scripts/a11y.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+/**
+ * @file a11y.js
+ * Contains a script that, when executed, will execute a11y linting tools
+ * against the storybook build.
+ */
+
+const R = require('ramda');
```

Comment:
I pulled in ramda because it's nice to have a lib of helpers, it makes some stuff easier. If ya'll would rather me not pull ramda in, or if you'd prefer lodash, lmk.